flow/shared/clickhouse/validation.go

```diff
@@ -82,24 +82,49 @@ func GetTableColumnsMapping(ctx context.Context, logger log.Logger, conn clickho
 	for _, table := range tables {
```

Comment:
`queryTables` could be kept to at most 200 capacity & reused if we had QuoteLiteral in chunking